src/xcode/ENA/ENA/Source/Client/HTTP Client/__tests__/HTTPClient+AppConfigTests.swift

```diff
@@ -97,6 +97,9 @@ final class HTTPClientAppConfigTests: XCTestCase {
 			case .failure(let error):
 				XCTFail("Country fetch error: \(error as NSError)")
 			case .success(let list):
+				
+				// As of 2020-09-16 the latest backend deployment returns zero (0) countries.
+				// Due to the current changes in INTEROP we should check this by end of Sept 2020!
```

Comment:
should we add a ticket so we do not forget about this?

Reply:
Added subtask.https://jira.itc.sap.com/browse/EXPOSUREAPP-2783